keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -190,8 +192,10 @@ spec:
   generateBucketName: [4]
   bucketClassName: [5]
   secretName: [6]
+  requestProtocol: [7]
+  requestAccessMode: {"ro", "rw"} [8]
```

Comment:
Shouldn't values be the same as in SCs and CSI:
```
The API access modes are:

- ReadWriteOnce – the volume can be mounted as read-write by a single node
- ReadOnlyMany – the volume can be mounted read-only by many nodes
- ReadWriteMany – the volume can be mounted as read-write by many nodes

In the CLI, the access modes are abbreviated to:

- RWO - ReadWriteOnce
- ROX - ReadOnlyMany
- RWX - ReadWriteMany

```

Reply:
I don't think so as Bucket's have a different list from volumes.  For one, all buckets are *many by their nature.

Reply:
I agree with _many_, but why would we create new names for read-write and read-only? Though, I think there is a `readOnly` field somewhere related to CSI.

Reply:
Because the *many and *once suffixes don't apply to a bucket context.  At most, there's RW, RO, WO.